src/App.tsx

```diff
@@ -1,9 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+interface Movie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies: Movie[] = moviesFromServer.filter((movie) => {
+    const trimmedQuery = query.trim().toLowerCase();
+    const { title, description } = movie;
+
+    return (
+      title.toLowerCase().includes(trimmedQuery)
+      || description.toLowerCase().includes(trimmedQuery)
+    );
+  });
+
+  const queryChange = (event: { target:{ value: string } }) => {
```

Comment:
Use react types for events
`React.ChangeEvent<HTMLInputElement>`

Reply:
fixed